pkg/controller/direct/backupdr/backupdrbackupvault_controller.go

```diff
@@ -250,6 +250,10 @@ func (a *BackupVaultAdapter) Delete(ctx context.Context, deleteOp *directbase.De
 	log.V(2).Info("deleting BackupVault", "name", a.id)
 
 	req := &pb.DeleteBackupVaultRequest{Name: a.id.String()}
+
+	if a.desired.Spec.IgnoreInactiveDatasources != nil && *a.desired.Spec.IgnoreInactiveDatasources {
+		req.Force = true
+	}
```

Comment:
Another option is to use the existing "cnrm.cloud.google.com/force-destroy" instead. But, I like this approach more considering the simplicity and specification of the API.   https://cloud.google.com/config-connector/docs/reference/annotations

Reply:
It looks like there are other deletion options besides "force" we might want to support in the future. Not sure we want to keep adding annotations.

[ignore_backup_plan_references](https://github.com/googleapis/googleapis/blob/bad8d92019660beca95396e71901d783fb8e1273/google/cloud/backupdr/v1/backupvault.proto#L823C8-L823C37)